tardis/adapters/sites/htcondor.py

```diff
@@ -44,7 +45,9 @@ async def condor_q(
         _job_id(resource.remote_resource_uuid) for resource in resource_attributes
     )
 
-    queue_command = f"condor_q {remote_resource_ids} -af:t {attributes_string}"
+    queue_command = (
+        f"condor_q {remote_resource_ids} -af:t {attributes_string}"  # noqa E231
```

Comment:
No related to the feature. Flake 8 complained about the missing space after the colon `-af: t`.